drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/projects_importer.rb

```diff
@@ -122,7 +150,7 @@ def upsert_projects
 
       check_columns(
         file: file,
-        expected_columns: ['ProjectID', 'OrganizationID', 'ProjectName', 'ProjectCommonName', 'OperatingStartDate', 'OperatingEndDate', 'ContinuumProject', 'ProjectType', 'HousingType', 'ResidentialAffiliation', 'TrackingMethod', 'HMISParticipatingProject', 'TargetPopulation', 'HOPWAMedAssistedLivingFac', 'PITCount', 'Walkin', 'DateCreated', 'DateUpdated', 'UserID', 'DateDeleted', 'ExportID'],
+        expected_columns: ['ProjectID', 'OrganizationID', 'ProjectName', 'ProjectCommonName', 'OperatingStartDate', 'OperatingEndDate', 'ContinuumProject', 'ProjectType', 'HousingType', 'RRHSubType', 'ResidentialAffiliation', 'TargetPopulation', 'HOPWAMedAssistedLivingFac', 'PITCount', 'DateCreated', 'DateUpdated', 'UserID', 'DateDeleted', 'ExportID', 'Walkin'],
```

Comment:
```suggestion
        expected_columns: [*GrdaWarehouse::Hud::Project.hmis_configuration(version: '2024').keys.map(&:to_s), 'Walkin'],
```

Could be nice to fix up - we should have done that the first time around